### ISSUE TITLE
What's the destiny of the old version?

### ISSUE THREAD

**Issue by joakim** [symbl-cc/symbl-data#164]
I prefer the old version and really hope it will be kept alive. What are your plans for it?

**Comment by alberic89** [comment]
For the moment, you can find it at [https://old.unicode-table.com](https://old.unicode-table.com)

**Comment by joakim** [comment]
I hope that moment lasts :)

**Comment by alberic89** [comment]
If it is removed, you can also [go back to the past](https://web.archive.org/web/20221225115600/https://unicode-table.com/en/).

**Comment by joakim** [comment]
That's true, it will just be a little slow. I hope they have archived everything.

**Comment by waldyrious** [comment]
I used (still use) the search function extensively in the old version. The archived version doesn't allow for that, except for queries that have been explicitly stored ([example](https://web.archive.org/web/20220527072027/https://unicode-table.com/en/search/?q=degree)). So I hope the old version will still remain live indefinitely so we don't have to rely on the archive.

**Comment by asanov** [comment]
The old version will be available at the https://old.unicode-table.com/

**Comment by waldyrious** [comment]
Just to be clear, are you planning to keep https://old.unicode-table.com available indefinitely, @asanov? Or is it possible that you'll retire it sometime in the future?

**Comment by asanov** [comment]
We won't update and maintance it. https://symbl.cc at the moment contains 2 times more information in terms of volume and accuracy. We will focus on it. Are there any reasons why you can't use the new version?

**Comment by waldyrious** [comment]
I need to test the new version more to be able to offer informed feedback. Maybe others in this thread can chime in in the meantime. For the time being, I updated my bookmarks to point to the old version, because I'm already familiar with the UI, but I suppose if all the data and functionality is still present in the new one, it may be just a matter of getting used to it.

**Comment by Nellty** [comment]
> https://symbl.cc at the moment contains 2 times more information in terms of volume and accuracy. We will focus on it. Are there any reasons why you can't use the new version?

I am not a designer, and not an active user of the website, but this new design is just bad. It feels like the things you could reach easy before are now hidden behind some distractions, and it's not only because I'm used to the old website, but because the new UI has some sort of lack of usability. Also:
- the font sizes **or** UI elements are vary from too big to too small at the same page;
- distracting animations;
- no OpenSearch;
- useless language switch button that haunts you everywhere;
- to type the name of the website in a search bar instead of "uni" that is on the same spot, now you need "sym" that is across the whole keyboard;

There were a problems with the old design (maybe?), but now it's became worse for no reason. It's just stupid

On top of that the search is looking broken on old.unicode-table. This returns nothing: https://old.unicode-table.com/en/search/?q=arrow

**Comment by asanov** [comment]
@Nellty Thank you for feedback! I can't stand idle chatter without any examples or facts.

> I am not a designer, and not an active user of the website, but this new design is just bad. It feels like the things you could reach easy before are now hidden behind some distractions.
Please list at least a few examples.

> the font sizes or UI elements are vary from too big to too small at the same page;
Can you attach a screenshot?

> distracting animations
Which exactly animations?

> no OpenSearch;
It has nothing to do with design. We will add this functionality in the near future.

> useless language switch button that haunts you everywhere;
Does it overlap the content or interfere in some other way?

**Comment by Basic7x7** [comment]
I think the main problem is the new start page. Think of a new user's first impression seeing
![old](https://user-images.githubusercontent.com/25965925/224161704-32ae8973-d2f0-4385-a34d-9b71de936fc9.png)
And now:
![new](https://user-images.githubusercontent.com/25965925/224159026-41e299a5-89e8-4b0d-bab7-ef379697bdcc.png)
My personal first impression to the new design was "I clicked the wrong site" and I closed it. After checking the domain I thought the domain was taken over and redirects to another site now.

To come to the design itself: Everything is too big and filled with random stuff like "Symbol of The Day". The only important things are the small plain text links on the right.

Also the new design implies a totally different target group. I think the old design was mainly used by programmers and web developers looking for Unicode codes of given characters and to get an overview where which characters are. In most cases you could do that even without leaving the main page.
On the other hand the new design's target group seems to be people not knowing how to use the builtin emoji keyboard.

**Comment by asanov** [comment]
@Basic7x7 Yes, you're absolutely right. 85% user of the website were people who were looking for Emoji and cool letters for their nicknames. So the new design is for them. This target group allow as to maintain, improove and develop the website thanks to monetization on banner network. BUT. We have kept all the content of the previous version, including UNICODE, improved its quality and continue to work on it.

**Comment by joakim** [comment]
@asanov I fully understand that prioritization.

To also cater to the remaining 15% pro users,  what about keeping the old domain name we all know by heart (unicode-table.com) and redirect it to [symbl.cc/en/unicode/table](https://symbl.cc/en/unicode/table/) once that's done? I assume it would look more like the old interface?

**Comment by asanov** [comment]
@joakim we can't redirect the whole old domain to this URL because in this case we will lose the audience of all other pages (more than 5 million on 16 languages) and our position in the search results. But we can redirect the old main page in Englsih unicode-table.com/en/ to the new Unicode Table page in English when it will be ready (a. in a month) symbl.cc/en/unicode/table/ and in other languages as well ofc.
Would it be helpful?

**Comment by joakim** [comment]
I see. I don't think that would be enough, I typically type `unicode-table.com` (well, it autocompletes to that) without `/en`.

I'd also be happy with a prominent link for pro users on the main page. Maybe below Symbol of The Day? I'm not sure which page it's best to link to, maybe this is an opportunity to rethink the interface for pro users?

Sorry if I'm just giving you more work. I don't mean to be ungrateful, I really appreciate the work that goes into maintaining this service that I've depended on for years :)

**Comment by asanov** [comment]
How do you usually use the website? Describe typical scenarios.
Question for all of you guys @joakim @Basic7x7 @Nellty @waldyrious @alberic89

**Comment by joakim** [comment]
1. Search: look up symbols or blocks (by name or by actual symbol)
2. Explore: look through blocks and sets
3. Tools: table generator, decoder, etc

More or less in that order. Typically with a goal to find technical/encoding information for symbols.

My main issue is with the homepage: all its distractions. The old design was more calm and to the point, this one is fun and chaotic. That's fine for your stated target audience, but distracting for professional users.

I just want to find the information I'm after, quick and easy, without unsolicited things grabbing my attention. What I mean when I think of a page for pro users: Focused, minimal, efficient.

Another issue is that technical information is now below the fold on symbol pages. It's what I'm typically after, so I'd prefer it to be visible without having to scroll. The extra effort required is annoying and adds up over time.

I do like the grouping of info tables into tabs, I think that's an improvement 👍

As far as I can tell, there's nothing I can't do on the new site that I could on the old. Although I prefer the old site's calm design and purple color scheme, I can live with the new design, even though I feel like I'm no longer the target audience when visiting the homepage.

**Comment by alberic89** [comment]
When I use the old site, it's mostly when I'm looking for a Unicode symbol related to a theme. I look for the table that may contain the symbol I'm looking for, I open it, I scroll, I look, I compare, I open several tabs. Once the symbol is chosen, I can immediately copy and paste it, and see at a glance the Unicode, HTML and CSS codes, the different correspondences between OS and browsers, etc.
In some cases I know what I want and look for the name in the search bar, but it's less frequent.
The design is simple and non-aggressive, not too heavy and the buttons and icons are well sized.

**Comment by avl93** [comment]
Old design looks line a normal tool for work I actually use on daily basis. New one - SCREAMS AT USER WITH **GIANT** LETTERS and a mEsS oF difFenEnT `$￥ℳ฿ტŁ$.`

**Comment by Epyxoid** [comment]
I mean the new site looks usable as well to be honest. But I don't think at all that this transition is intentional. The page design has day 1 feeling. It was either a forced or a dumb choice.

**Comment by mksplg** [comment]
I'm not opposed to the new design in general but it's weird that everything is so giant, especially on widescreen monitors. 

![grafik](https://user-images.githubusercontent.com/1591254/237036602-f7eda8a3-82d6-4a4f-b63d-633514b3eed8.png)

It's even going so far that you can't see the whole tagline or search preview field without scrolling just so the site name letters are more than 5cm high on my screen. Looks like the whole layout is messed up because the CSS is not loading.

![grafik](https://user-images.githubusercontent.com/1591254/237037588-d8307799-9ec9-4164-be52-32d227151759.png)
